### ISSUE TITLE
FKinSpace and FKinBody throw "warning: colon arguments should be scalars" in Octave

### ISSUE THREAD

**Issue by gelanchez** [NxRLab/ModernRobotics#37]
F​unctions FKinSpace and FKinBody give the following message when executed:
_warning: colon arguments should be scalars_
w​hen executed under Octave, due to the fact that function size returns a vector containing the number of rows and number of columns. Not sure under MATLAB. I believe in their code, they should use:
`for i = 1: size(thetalist)(1)`
if thetalist is a column vector, i​nstead of:
`for i = 1: size(thetalist)`

**Comment by HuanWeng** [comment]
Yes you are right. I'm not familiar with Octave but I think `size(thetalist, 1)` is better in MATLAB. We will change this in our next version.
